### PR TITLE
test(e2e): review e2e

### PR DESCRIPTION
**Pull request by zoesteinkamp**
These are adjustments made to our e2e tests

### CODE REVIEW

**Comment by 121watts** [comment]
@zoesteinkamp you want to rebase master and give these a whirl?

File: cypress/e2e/orgs.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const secondOrg = 'Second Org'
 
 // NOTE: this is dying for no reason in circleci
```
Comment (121watts) [inline cypress/e2e/orgs.test.ts:3]: Probably can remove this comment

File: cypress/e2e/queryBuilder.test.ts [hunk 2]
```diff
@@ -82,6 +83,8 @@ describe('The Query Builder', () => {
       cy.getByTestID('nav-item-dashboards').click()
 
       cy.contains('Basic Ole Dashboard').click()
+      cy.getByTestID('giraffe-layer-line').should('exist')
+      cy.getByTestID('cell-context--toggle').should('exist')
```
Comment (121watts) [inline cypress/e2e/queryBuilder.test.ts:87]:
```suggestion
      cy.getByTestID('giraffe-layer-line').should('be.visible')
      cy.getByTestID('cell-context--toggle').should('be.visible')
```

**Review by 121watts — APPROVED** [review]
Some tiny things to clean up.  Largest issue is the possible race condition on bucket deletion.

File: cypress/e2e/buckets.test.ts [hunk 3]
```diff
@@ -147,12 +146,6 @@ describe.skip('Buckets', () => {
       cy.getByTestID(`context-delete-menu ${bucket1}`).click()
       cy.getByTestID(`context-delete-bucket ${bucket1}`).click()
 
-      // normally we would assert for empty state here
-      // but we cannot because of the default system buckets
-      // since cypress selectors are so fast, that sometimes a bucket
-      // that is deleted will be selected before it gets deleted
-      cy.wait(10000)
-
       cy.getByTestID(`bucket--card--name ${bucket1}`).should('not.exist')
```
[the remark this answers is not in the document]
Comment (zoesteinkamp) [reply cypress/e2e/buckets.test.ts:149]: ive commented this out so i can address it in a separate pr and shipping this larger one out

**Pull request merged**